### PR TITLE
Add semver validator tool release notes

### PR DESCRIPTION
**Pull request by NipunaRanasinghe**
## Purpose
$subject.

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md [hunk 1]
```diff
@@ -115,6 +115,46 @@ To view bug fixes, see the [GitHub milestone for 2201.2.0 (Swan Lake)](https://g
 
 #### New features
 
+##### SemVer validator CLI tool [Experimental] 
+Introduced the `bal semver` CLI command, which attempts to validate <a href="https://semver.org/">SemVer</a> compatibility of the local package changes against any previously published version(s) in Ballerina Central. Currently the tool can be used to
```
Comment (praneesha) [inline release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md:119]:
```suggestion
Introduced the `bal semver` CLI command, which attempts to validate <a href="https://semver.org/">Semantic Versioning</a> compatibility of the local package changes against any previously published version(s) in Ballerina Central. Currently, the tool can be used to: 
```

File: release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md [hunk 2]
```diff
@@ -115,6 +115,46 @@ To view bug fixes, see the [GitHub milestone for 2201.2.0 (Swan Lake)](https://g
 
 #### New features
 
+##### SemVer validator CLI tool [Experimental] 
+Introduced the `bal semver` CLI command, which attempts to validate <a href="https://semver.org/">SemVer</a> compatibility of the local package changes against any previously published version(s) in Ballerina Central. Currently the tool can be used to 
+- list-down the source code differences(along with compatibility impact) between the local and any published versions in Ballerina central
```
Comment (praneesha) [inline release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md:120]:
```suggestion
- list down the source code differences (along with its compatibility impact) between the local and any published versions in Ballerina central
```

File: release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md [hunk 3]
```diff
@@ -115,6 +115,46 @@ To view bug fixes, see the [GitHub milestone for 2201.2.0 (Swan Lake)](https://g
...
+- suggest the new package version based on the compatibility impact of source code changes
+
+The below example demonstrates semver validator tool functionalities on the `ballerina/io` package.
```
Comment (praneesha) [inline release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md:123]:
```suggestion
The example below demonstrates the semver validator tool functionalities on the `ballerina/io` package.
```

File: release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md [hunk 4]
```diff
@@ -115,6 +115,46 @@ To view bug fixes, see the [GitHub milestone for 2201.2.0 (Swan Lake)](https://g
...
+
+i. version suggestions
```
Comment (praneesha) [inline release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md:125]:
```suggestion
i. version suggestions
```
Shall we make these sub headings?

File: release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md [hunk 5]
```diff
@@ -115,6 +115,46 @@ To view bug fixes, see the [GitHub milestone for 2201.2.0 (Swan Lake)](https://g
...
+```
+$bal semver             
+checking for the latest compatible release version available in central...
+
+current version: 1.2.2-SNAPSHOT
+compatibility impact (compared with the release version '1.2.1'): backward-incompatible changes detected
+suggested version: 2.0.0
+```
+ii. version suggestions with the list of detected source code changes
```
Comment (praneesha) [inline release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md:134]:
```suggestion
ii. version suggestions with the list of detected source code changes
```
Shall we make these sub headings?

File: release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md (hunk 4 above)
Comment (NipunaRanasinghe) [reply release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md:125]: Actually these are not sub heading of the semver tool topic and just example scenarios. so shall we keep as it is?

File: release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md (hunk 5 above)
Comment (NipunaRanasinghe) [reply release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md:134]: https://github.com/ballerina-platform/ballerina-release/pull/2070#discussion_r929703854

File: release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md (hunk 4 above)
Comment (praneesha) [reply release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md:125]: +1 to keep as normal bullet points (not numbered).

File: release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md (hunk 5 above)
Comment (praneesha) [reply release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md:134]: +1 to keep as normal bullet points (not numbered).

File: release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md (hunk 4 above)
Comment (NipunaRanasinghe) [reply release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md]: Added with https://github.com/ballerina-platform/ballerina-release/pull/2070/commits/ca7bd413e820b182031b23a29e2bed6514e19eff

File: release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md (hunk 5 above)
Comment (NipunaRanasinghe) [reply release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md]: Added with https://github.com/ballerina-platform/ballerina-release/pull/2070/commits/ca7bd413e820b182031b23a29e2bed6514e19eff

**Review by hevayo — APPROVED** [review]

**Review by praneesha — APPROVED** [review]

**Pull request merged**